src/diffpy/pdfgui/gui/fitnotebookpanel.py

```diff
@@ -20,12 +20,17 @@
 from diffpy.pdfgui.gui.pdfpanel import PDFPanel
```

Comment:
Did you used wxGlade to generate the GUI codes? If so, did you use the most updated version?